content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java

```diff
@@ -171,6 +171,8 @@
 	/** Tool property to enable Drag and Drop uploads in a per-tool basis */
 	private static final String TOOL_PROP_DRAGNDROP_ENABLED = "content.upload.dragndrop";
 
+	private static final String LABEL_VERSION = "label.version";
```

Comment:
This was made dynamic with the thought that, if in future {label.version} is worded differently ,then it requires changing in 2 places manually. 

Do you think its not worth it.?